.github/workflows/ci.yml

```diff
@@ -76,9 +76,10 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install Flutter
-        uses: subosito/flutter-action@v1
+        uses: subosito/flutter-action@v2
```

Comment:
would it make sense to also use the v2 version of this action for the example job at the end of this file?